tests/test_matrix_utils_sync_bootstrap_login.py

```
@@ -15,6 +15,7 @@
 
 TEST_CREDS_PATH = str(Path(tempfile.gettempdir()) / "creds.json")
 TEST_E2EE_STORE_PATH = str(Path(tempfile.gettempdir()) / "e2ee_store")
+TEST_PASSWORD = "password"
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to: "TEST_PASSWORD"

(S105)